src/ansys/units/cfg.yaml

```diff
@@ -168,11 +168,11 @@ base_units:
     si_offset: 273.15
   F:
     type: TEMPERATURE
-    si_scaling_factor: 0.55555555555
+    si_scaling_factor: 0.5555555555555555
```

Comment:
Alternatively we could write 5/9 and call `eval()` on each value in the Python code.

Reply:
@seanpearsonuk It gives incorrect conversion.

![image](https://github.com/user-attachments/assets/44e99770-facd-4765-9c2f-a02c5d1bd942)